scripts/ci/mutation_triage_queue.py

```
@@ -0,0 +1,197 @@
+
+    end = len(lines)
+    for index in range(start + 1, len(lines)):
+        if opener.match(lines[index]):
+            end = index
+            break
+
+    window = lines[start:end]
+    # Keep entries readable; a very long function is a smell worth seeing, but
+    # the queue should not become a source dump.
+    if len(window) > 60:
+        window = window[:60] + ["    // … truncated, see the file for the rest"]
```

Ruff (0.15.21)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)